MetabolicAtlas/MetabolicAtlas
# fix: rename external db endpoint and related variable names etc

**Related issue(s) and PR(s)**  
This PR closes #995 and relates to #994 .


The external-db endpoint has been renamed and variables etc related to this endpoint have been updated to reflect the new naming

**Type of change**  

 
**List of changes made**  

- Rename the endpoint `/external-db/:dbName/:externalId` to `/identifier/:dbName/:externalId`
- Change variable names, tests and file names 
- Add the new endpoint to Swagger

**Testing**  

- Try the new endpoint, e.g. http://localhost/api/v2/identifier/MetabolicAtlas/MAM10036c
- Check the usages of identifier in front end e.g http://localhost/identifier/HMR%202.0/HMR_5934 and
http://localhost/identifier/MetabolicAtlas/MAM02877e
- Go to http://localhost/api/v2/ and make a query to the Identifier endpoint
- Run `ma-exec api yarn test` and check that the identifier and crossReference tests pass

**Further comments**  

- The Swagger file may need some changes (e.g. description of the endpoint)

**Definition of Done checklist**
